api/src/main/java/marquez/service/models/Dataset.java

```diff
@@ -61,6 +62,7 @@
   @Nullable private final Instant lastModifiedAt;
   @Nullable private final String description;
   private final Optional<UUID> currentVersionUuid;
+  @Nullable Facets facets;
```

Comment:
It would be nice if we can just use `Optional` fields and let Jackson handle the de/serialization